frontend/src/components/dialogs/MemberDialog.vue

```diff
@@ -229,11 +272,14 @@ export default {
       return undefined
     },
     serviceAccountNames () {
-      const serviceAccounts = filter(this.memberList, ({ username }) => isServiceAccount(username))
-      return map(serviceAccounts, ({ username }) => serviceAccountToDisplayName(username))
+      const serviceAccounts = filter(this.memberList, ({ username }) => isServiceAccountUsername(username))
+      return map(serviceAccounts, 'username')
+    },
+    serviceAccountNamesWithoutPrefix () {
```

Comment:
create a new getter
```js
 memberUsernames () {
   return map(this.memberList, 'username') 
 }
```
and use it `projectUsernames` and `serviceAccountUsernames`:
```js
...
serviceAccountUsernames () {
  return filter(this.memberUsernames, isServiceAccountUsername)
},
projectUsernames () {
  return filter(this.memberUsernames, negate(isServiceAccountUsername))
}
...
```